### PR TITLE
UPDATED: qrUrl on provider to be optional

### PR DESCRIPTION
**Pull request by jeongmincho**
non-optional qrUrl made breaking change on extension, making the parameter optional.

### CODE REVIEW

File: src/provider/CoinbaseWalletProvider.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ const HAS_CHAIN_OVERRIDDEN_FROM_RELAY = "HasChainOverriddenFromRelay";
 export interface CoinbaseWalletProviderOptions {
   chainId?: number;
   jsonRpcUrl: string;
-  qrUrl: string | null;
+  qrUrl?: string | null;
```
Comment (bangtoven) [inline src/provider/CoinbaseWalletProvider.ts:53]: isn’t it supposed to be something like
```suggestion
  qrUrl?: string = undefined;
```
and come as last param or something?

**Review by bangtoven — COMMENTED** [review]
@jeongmincho thanks for quick turnaround. one question on js syntax as i am a js noob

**Review by cb-jake — DISMISSED** [review]

**Comment by cb-heimdall** [comment]
Review Error for cb-jake @ 2022-04-26 23:47:49 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

File: src/provider/CoinbaseWalletProvider.ts (hunk 1 above)
Comment (bangtoven) [reply src/provider/CoinbaseWalletProvider.ts:53]: nvm. confirmed it works. i think i was confused with optional function param.

**Review by bangtoven — APPROVED** [review]

File: src/provider/CoinbaseWalletProvider.ts (hunk 1 above)
Comment (erin-at-work) [reply src/provider/CoinbaseWalletProvider.ts:53]: Does it need `null` at all?

**Pull request merged**